legal_doc_processing/press_release/defendant/defendant/utils_defendant.py

```diff
@@ -0,0 +1,267 @@
+from transformers import pipeline
+nlp2 = pipeline("question-answering")
+
+import pandas as pd
+import numpy as np 
+import os
+import re
+import requests
+
+def _del_dummy_breaklines(txt: str):
+
+  new_txt = (
+      txt.replace("\n.", "$$$$")
+      .replace("\n", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("$$$$", "\n")
+  )
+
+  return new_txt
+
+
+def _del_bouble_breaks_and_spaces(txt: str) -> str:
+
+  new_txt = (
+      txt.replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("  ", " ")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
+      .replace("\n\n", "\n")
```

Comment:
can be replaced by  loop or a better regex

Reply:
Il s'agissait d'une ancienne fonction du github. Du coup j'avais gardé comme tel. Mais là j'ai utilisé une expression régulière qui m'a permis aussi de me passer d'elle